test/integration/scheduler_perf/label_selector_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package benchmark
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestLabelFilter(t *testing.T) {
+	empty := ""
+	performance := "performance"
+	fastPerformance := "performance,fast"
+	notFast := "-fast"
```

Comment:
I'd like to cover a "conflicting" case, like:

```
performanceNoPreemption := "+performance,-preemption"
```

so suppose we have a test case tagged with "performance,preemption", this test case would be filtered out.

Reply:
I've [added `"+performance,-fast"`](https://github.com/kubernetes/kubernetes/compare/01c5f5466078d71b8b9d0bc901ac9cb01e94cb0a..550d4c00746721ba06b0756d4dde5857d3dd7ea8). When checking that against `["performance","fast"]`, that workload gets filtered out, as you said.